front/src/components/UI/CompanyView/CompaniesSector.tsx

```diff
@@ -0,0 +1,22 @@
+import CompanyBlock from "./CompanyBlock";
+
+const Companies = (props: any) => {
```

Comment:
zamiast any zdefiniuj proszę interfejs (tak jak jest to zrobione  np w Button.tsx). Gdy dajemy any, to niestety kompilator przestaje sprawdzać czy na pewno odwołujemy się do prawidłowych danych. Szczególnie problematyczne może się to okazać w przyszłości, szczególnie jeśli ktoś inny będzie próbował analizować kod